test/lenses.js

```diff
@@ -41,16 +56,24 @@ describe('view, over, and set', function() {
 
   they('may be applied to composed lenses', function() {
     var streetLens = R.compose(addressLens, headLens);
+    var dogLens = R.compose(R.lensPath(['pets']), R.lensPath(['dog']));
+
+    eq(R.view(dogLens, alice), R.view(R.lensPath(['pets', 'dog']), alice));
```

Comment:
This is not a test of the public interface of `lensPath`, only of how it relates to another function.  Much more appropriate would be to note

```js
eq(R.view(dogLens, alice), 'joker');
```